packages/web-app-files/src/views/PublicFiles.vue

```diff
@@ -108,10 +104,10 @@ const unauthenticatedUserReady = async (router, store) => {
   // ocis at the moment is not able to create archives for public links that are password protected
   // till this is supported by the backend remove it hard as a workaround
```

Comment:
I guess the fact that the archiver doesn't support password protected public link is not solved, so this still needs to be done. But is it tracked in a reva ticket? Would be helpful to have the issue link here, so that it's easier to validate if the check can be removed.

Reply:
https://github.com/owncloud/web/issues/6515

Reply:
Thank you! Can you please add the issue link to the comment in the code?

Reply:
Oh, you did. sorry...